tools/local-env/install-node-nvm.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+NVM_VERSION=`curl -Ls -w %{url_effective} -o /dev/null https://github.com/creationix/nvm/releases/latest | rev | cut -d '/' -f 1 | rev`
```

Comment:
Should probably be `https://github.com/nvm-sh/nvm/releases/latest`. NVM seems to have moved. The current URL is redirecting correctly but this may cause confusion at a later point. Same for the other URLs in this file.

Reply:
We should update in Gutenberg as well then :)